grpc/server/plugins/action/action.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package dronecorerpc;
+
+service ActionRPC {
+    rpc Arm(ActionEmpty) returns(ActionResult) {}
+    rpc TakeOff(ActionEmpty) returns(ActionResult) {}
+    rpc Land(ActionEmpty) returns(ActionResult) {}
+}
+
+// FIXME these different Empty types are ugly
```

Comment:
I think there is a protobuf empty that we should reuse: `import "google/protobuf/empty.proto";`.